AutoGear.lua

```diff
@@ -3935,10 +3939,13 @@ function AutoGearGetBest1hPairing(info)
 end
 
 function AutoGearTooltipHook(tooltip)
+	-- Added nil check to avoid errors which were happening on GetItem() call
+	if tooltip ~= GameTooltip or tooltip == nil then return end
 	if (not AutoGearDB.ScoreInTooltips) then return end
 	if (not AutoGearCurrentWeighting) then AutoGearSetStatWeights() end
 	local name, link = tooltip:GetItem()
-	local equipped = tooltip:IsEquippedItem()
+	-- Changed to this call which I found in Blizzard code while researching and seems to work well
+	local equipped = IsEquippedItem(name)
```

Comment:
This can be changed back, it was a proactive guess since I was having issues with the Tooltip:GetItem() call as well, I made change thinking it would help with the issues.

Reply:
Sounds good.